### ISSUE TITLE
In-place operation of tensor in SynthesisBlock

### ISSUE THREAD

**Issue by qrzou** [NVlabs/stylegan2-ada-pytorch#115]
Hello,
I found there's a in-place operation .add_() in SynthesisBlock when add two imgs together.
Will it lead to any gradients mistakes? Or it's used by design?

https://github.com/NVlabs/stylegan2-ada-pytorch/blob/d4b2afe9c27e3c305b721bc886d2cb5229458eba/training/networks.py#L415
